File: extract_pdf.js
const fs = require('fs');
const pdf = require('pdf-parse');

let dataBuffer = fs.readFileSync('y:\\takip\\Hammer_Kullanimi.pdf');

pdf(dataBuffer).then(function(data) {
    fs.writeFileSync('y:\\takip\\extract_pdf_out.txt', data.text);
    console.log('Success');
}).catch(function(error){
    console.error(error);
});

File: extract_pdf.py
import fitz

doc = fitz.open("y:/takip/Hammer_Kullanimi.pdf")
text = ""
for page in doc:
    text += page.get_text()

with open("y:/takip/extract_pdf_out.txt", "w", encoding="utf-8") as f:
    f.write(text)

print("Success")
